src/api/content-strings/XLSX.ts

```diff
@@ -102,7 +102,7 @@ export const XLSX = {
     [BROKER_ADDRESS]: 'Broker address',
     [BROKER_EMAIL]: "Broker's email address",
     [BUYER_COMPANY_NAME]: 'Buyer company name or organisation',
-    [BUYER_CONTACT_DETAILS]: 'Buyer contact details',
+    BUYER_CONTACT_DETAILS: 'Buyer contact details',
```

Comment:
is there a reason we are not using a constant for this?

Reply:
Yes, It's not a field ID, it's an XLSX specific field. We have this for some other fields specific to the XLSX, e.g `SOMEONE_ELSE`